drafts/use-cases/1.entry-point.md

```diff
@@ -0,0 +1,108 @@
+## Entry point
+
+### Story
+As an application user
+I want my application to consume an entry point for the Web API by providing a base URL
+So I can start managing my events.
+
+### Usage
+```javascript
+var client = new HydraClient();
+var data = client.get("/api");
```

Comment:
But what if the client gets _http://www.example.com/api_ - which is already an entry point. This way you won't fetch any API documentation headers (I wouldn't expect those here).
From my experience APIs (_http://www.example.com/api_ in this example) are usually added to the existing site (_http://www.example.com/_) and the main Url doesn't care about the API.
I believe we shall have these scenarios covered in the spec.